tests/node/test_wakunode_sharding.nim

```diff
@@ -431,7 +432,7 @@ suite "Sharding":
         let clientResult = await clientHandler.waitForResult(FUTURE_TIMEOUT)
         assertResultOk(clientResult)
 
-      xasyncTest "store (automatic sharding filtering)":
+      asyncTest "store (automatic sharding filtering)":
```

Comment:
should we split this change into a different PR?

Reply:
I don't mind either way. If you rather have it separate I'll move it.